examples/gno.land/r/demo/tests/tests.gno

```diff
@@ -78,8 +78,8 @@ func GetPrevRealm() std.Realm {
 	return std.PrevRealm()
 }
 
-func GetPSubtestsPrevRealm() std.Realm {
-	return subtests.GetPrevRealm()
+func GetRSubtestsPrevRealm() std.Realm {
```

Comment:
This method name was incorrect, it's not calling `p/tests/subtests` but `r/tests/subtests`. Fortunately, the other tests that are using it weren't misled.